recipes/certificate_server.rb

```diff
@@ -4,10 +4,10 @@
 #
 # Copyright (c) 2015 The Authors, All Rights Reserved.
 
-master_servers = node['cookbook-openshift3']['master_servers']
-certificate_server = node['cookbook-openshift3']['certificate_server'] == {} ? node['cookbook-openshift3']['master_servers'] : master_servers + [node['cookbook-openshift3']['certificate_server']]
+server_info = OpenShiftHelper::NodeHelper.new(node)
+is_certificate_server = server_info.on_certificate_server?
 
-if certificate_server.find { |server_master| server_master['fqdn'] == node['fqdn'] }
```

Comment:
Need to be re-worked as it now excludes the peer servers which also need to get the oc client installed